### PR TITLE
MCR-1950 added schema.org support

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-1950).

### CODE REVIEW

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 24b04d1ad89093d57131abc8fd142a2af1cae0bb! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/3963-pull_request-694

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on ade7812cf78e444f413c6c680a50d00b68181881! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/3966-pull_request-694

File: mycore-base/src/main/java/org/mycore/frontend/cli/MCRClassification2Commands.java [hunk 1]
```diff
@@ -539,6 +539,7 @@ public static void checkClassification(String id) {
         }
     }
 
+
```
Comment (kkrebs) [inline mycore-base/src/main/java/org/mycore/frontend/cli/MCRClassification2Commands.java]: This file should not be part of this PR. Please reset changes to HEAD.

**Review by kkrebs — APPROVED** [review]
